src/App.jsx

```diff
@@ -45,43 +50,63 @@ const App = () => {
             <Route exact path="/">
               <Login />
             </Route>
+          <IonTabs>
+            <IonRouterOutlet>
+              <Route exact path="/home" component={Home} />
+              <Route exact path="/profile/:username" component={Profile} />
+              <Route exact path="/profile" component={Profile} />
+              <Route exact path="/profile/edit" component={EditProfile} />
+              <Route exact path="/settings" component={Settings} />
+              <Route exact path="/collections/:collectionId">
+                <Collection />
+              </Route>
+              <Route exact path="/collections/:collectionId/edit">
+                <EditCollection />
+              </Route>
+              <Route exact path="/collections/add">
+                <AddCollection />
+              </Route>
+              <Route exact path="/collections/:collectionId/items/:itemId">
+                <Item />
+              </Route>
+              <Route exact path="/collections/:collectionId/items/:itemId/edit">
+                <EditItem />
+              </Route>
+              <Route exact path="/collections/:collectionId/items/add">
+                <AddItem />
+              </Route>
+              {/* Test Route, DELETE */}
+              <Route exact path="/test/:collectionId/:itemId">
+                <AddCollection />
+              </Route>
```

Comment:
Minor: I think we can probably standardise how we want to display routes. Also, I'm thinking if there's a need to handle `/profile/:username` and `/profile` (my own) separately?